### ISSUE TITLE
ONE-vscode logo Design Contest!

### ISSUE THREAD

**Issue by llFreetimell** [Samsung/ONE-vscode#1239]
## :mega:  ONE-vscode logo Design Contest

We do not have any logo for ONE-vscode.
So let's create an awesome logo for ONE-vscode!
Anyone who are interested in this contest are welcomed :D

### Participation

- Anyone can participate.
- Make and upload your logos in this issue as comments.

### Artifacts

Following **artifacts would be merged with Apache-2.0 license**, which is ONE-vscode's license.

- svg file with original size
- png file with original size
- png file with 128x128 pixels size
- logo guideline written by English (pdf)

### Voting Rules

- Anyone can vote.
- Only 👍 are counted (:smile: , :heart: , :tada: , etc are not counted).
- You can vote as many candidates as you want.

### Winner Selection

- Basically, candidate with the largest number of :+1: will be winner.
  - If there are multiple candidates with the same number of :+1: , maintainers can choose a winner among them.
- However, other candidate with the second(or more less) largest number of :+1: can be winner when
  - there are any illegal actions such as copyright invasions, controversies, abusing, etc.
  - voters intentionally voted just for fun without any consideration for ONE-vscode.

### Schedule

- 8th Sep : Final call for uploading logo image
- 14th Sep : Final call for voting
- 15th Sep : Notification for a winner
- 23rd Sep : Final call for merging final artifacts

---

/cc @Samsung/one-vscode

**Comment by dayo09** [comment]
Q. Could you provide ONE logo files so that I can put subtle modification and submit it as a candidate?

**Comment by llFreetimell** [comment]
@dayo09 You can get it at https://github.com/Samsung/ONE/tree/master/logo/images :)
But maybe following is what you want to do...?

### Candidate 1

![1659926122988-0](https://user-images.githubusercontent.com/24720192/186628793-8adbade9-d256-4eb6-9da4-5601d2b29392.png)

### Candidate 2

![1659926122988-1 (002)](https://user-images.githubusercontent.com/24720192/186628937-0324b9e1-294a-42d4-a467-cf4b3274f711.png)

---

To prevent misunderstanding...!
Few weeks ago, I requested above images to @nkoong who is author of ONE logo.
It was because I didn't plan this logo contest and I was just to use those images for 0.3.0 release.
However, @lemmaa suggested this contest and expected that some more casual and brand new logo would be generated.
So I created this contest and then preparing to upload above images by @nkoong.

**Comment by lemmaa** [comment]
@llFreetimell , how about keeping one candidate per comment for easy voting? :)

**Comment by dayo09** [comment]
Thanks :-D

I tried to put vscode logo instead of 'vscode' characters, likewise 'git codespace' extension does, but it seems vscode team doesn't permit.


#### ref1. VSCODE logo guidelines
![image](https://user-images.githubusercontent.com/17171963/186804846-3bf49f8a-403e-4aa0-917a-13a9c8f4bbe8.png)

https://code.visualstudio.com/brand

#### ref. git codespace icon
![image](https://user-images.githubusercontent.com/17171963/186805024-31103a4c-f15c-43d4-8be3-0ca4dc09843e.png)

**Comment by llFreetimell** [comment]
### Announcement for Postponement

As there are no more than one participant, deadline is postponed for a while.
I will re-announce modified schedule in the near future.

**Comment by jyoungyun** [comment]
I want to add the logo image before v0.5.0 release because we will publish ONE-vscode extension to public marketplace.

So I tested several logo files.

- Use `samsungblue` logo image
![samsungblue](https://user-images.githubusercontent.com/7223627/213085632-b121c920-5076-43e6-9ce0-539db0b73b8f.png)
- Use `white` logo image
![white](https://user-images.githubusercontent.com/7223627/213085637-c6ec6fa1-d229-4d62-a841-71fe509a153c.png)
- Use `samsungblue` logo with white circle background image
![samsungblue_1](https://user-images.githubusercontent.com/7223627/213085633-37dd47ed-3f6e-48dc-948b-e7f5e397f4ca.png)
- Use `samsungblue` logo with vscode icon
![samsungblue_2](https://user-images.githubusercontent.com/7223627/213089215-341ccb7c-eb64-41e5-9912-6f24e8e8a962.png)

Please let me know your opinion. Or if you have a good idea or logo file, please share it.

_If you don't have any opinions, I will use the `samsungblue logo with white circle background image` as a logo file._

/cc @Samsung/one-vscode

**Comment by llFreetimell** [comment]
According to https://github.com/Samsung/ONE-vscode/issues/1239#issuecomment-1227981386, `Use samsungblue logo with vscode icon` is forbidden :)

I want to vote for option 3, `Use samsungblue logo with white circle background image` :D

**Comment by jyoungyun** [comment]
> According to https://github.com/Samsung/ONE-vscode/issues/1239#issuecomment-1227981386, `Use samsungblue logo with vscode icon` is forbidden :)

Ah, I misunderstood that comment. I removed the fourth one. :)

**Comment by dayo09** [comment]
+1 for option 3 😄

**Comment by jyoungyun** [comment]
According to the result of voting, I will upload the option 3 image soon.
